pkg/skuba/actions/cluster/init/init.go

```diff
@@ -141,10 +141,10 @@ func isAddonRequired(addon addons.Addon, initConfiguration InitConfiguration) bo
 }
 
 func writeScaffoldFiles(initConfiguration InitConfiguration) error {
-	scaffoldFilesToWrite := criScaffoldFiles["criconfig"]
+	scaffoldFilesToWrite := CriScaffoldFiles["criconfig"]
 	kubernetesVersion := initConfiguration.KubernetesVersion
 	if kubernetesVersion.Minor() < 18 {
```

Comment:
This condition can be removed, we won't provision 1.17 with this branch of skuba.